src/Renci.SshNet/Abstractions/ThreadAbstraction.cs

```diff
@@ -36,16 +29,10 @@ public static void ExecuteThreadLongRunning(Action action)
         /// <param name="action">The action to execute.</param>
         public static void ExecuteThread(Action action)
         {
-#if FEATURE_THREAD_THREADPOOL
```

Comment:
I'm afraid you'll need to revert these changes. ThreadPool is not available for **UAP10.0** TFM (see https://github.com/sshnet/SSH.NET/wiki/Conditional-compilation-symbols for more information).